docs/guide/web/default-theme.md

```diff
@@ -263,6 +263,44 @@ Customize labels used in the templates, useful for translating the interface for
     }
     ```
 
+### information <Badge text="added in v1.6.0" type="tip"/>
+
+Define the content of the navigation’s “information” block (see [nav](#nav) option above). This can be used to display a build date or the library version for example.
+
+```js
+{
+    information: [
+        {
+            label: 'Version',
+            value: require('./package.json').version,
+        },
+        {
+            label: 'Built on',
+            value: new Date(),
+            type: 'time', // Outputed as a <time /> HTML tag
```

Comment:
```suggestion
            type: 'time', // outputs a <time /> HTML tag
```
actually not sure what the correct past tense of output is but outputed feels weird - i think present tense makes a bit more sense.

Reply:
✅